src/styles/main.scss

```diff
@@ -1,3 +1,37 @@
+@import "./variables";
+@import "./calendar";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar__day {
+  width: $square_side;
+  height: $square_side;
+  border: 1px solid black;
+  background-color: $grey;
+  text-align: center;
+  transition-duration: 500ms;
```

Comment:
Just for your info, better to use 'transition: transform 0.5s;' instead of separate transition-duration and transition-property.